.travis.yml

```diff
@@ -1,18 +1,11 @@
-osx_image: xcode7.1
+osx_image: xcode7.2
 language: objective-c
-before_install:
-  - brew update
-  - brew outdated xctool || brew upgrade xctool
-  - gem install cocoapods
-  - pod install
```

Comment:
Actually looks like this may run automatically now: https://docs.travis-ci.com/user/languages/objective-c/#Dependency-Management